src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -3849,6 +3849,141 @@ public void MethodUpdate_LambdaParameterRefnessInBody()
                 "Update [public void M(int a) { f((ref int b) => b = 1); }]@10 -> [public void M(int a) { f((out int b) => b = 1); }]@10");
         }
 
+        [Fact]
+        public void MethodUpdate_LocalFunctionAddAttribute()
```

Comment:
>MethodUpdate_LocalFunctionAddAttribute [](start = 20, length = 38)

These tests should rather be in StatementEditingTests (near LocalFunctions_InExpressionStatement). After all, local functions are not "top-level". I think the above 2 tests are validating that update to a local function / lambda signature is not treated as top-level update.